app/routes/_gcn.news._index.mdx

```diff
@@ -24,6 +24,27 @@ import { Anchor } from '~/components/Anchor'
 
 <Collection>
 
+## 2024
+
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'February 28, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>New Announcement Feature, Code of Conduct, Circular Revisions</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        - **Announcements from the GCN Team to our User Community:**  These announcements will contain information on new GCN features, changes to GCN Notices schema, new GCN Notices topics, and planned system outages. All GCN users are automatically subscribed to GCN Announcements.  You can modify your subscription at any time by following the personalized link at the bottom of announcement emails, or logging into your account, selecting "Email Notifications" in the user dropdown menu and toggling the announcements on/off.
+
+        - The [**GCN Code of Conduct**](https://gcn.nasa.gov/docs/code-of-conduct) describes policies for community behavior as well as mechanisms for retractions, errata, reproduction of GCN Notices and Circulars, enforcement, and acknowledgements.  These policies represent the informal standards that have supported the thriving GCN Community over the last 32 years, but benefits from formal description.
```

Comment:
```suggestion
        - The [**GCN Code of Conduct**](https://gcn.nasa.gov/docs/code-of-conduct) describes policies for community behavior as well as mechanisms for retractions, errata, reproduction of GCN Notices and Circulars, enforcement, and acknowledgements.  These policies represent the informal standards that have supported the thriving GCN Community over the last 32 years, but benefit from formal description.
```

If they are informal, then how can they have enforcement mechanisms?

Reply:
I suppose they formalize the informal standards of the past